da_ddt-ud-train.conllu

```diff
@@ -910,7 +910,7 @@
 20	loko-føreren	loko-fører	NOUN	_	Definite=Def|Gender=Com|Number=Sing	19	obj	_	_
 21	som	som	ADP	_	PartType=Inf	23	case	_	_
 22	den	den	DET	_	Gender=Com|Number=Sing|PronType=Dem	23	det	_	_
-23	skyldige	skyldig	ADJ	_	Definite=Def|Degree=Pos|Number=Sing	19	amod	_	_
+23	skyldige	skyldig	ADJ	_	Definite=Def|Degree=Pos|Number=Sing	19	nmod	_	_
```

Comment:
Agreed. I should have checked more carefully. Does this mean that "som" should be mark rather than case (even though the predicate is nominal)?

Reply:
> Does this mean that "som" should be mark rather than case (even though the predicate is nominal)?

I'm not sure. On one hand, in our taxonomy we present `case` as belonging to nominals and `mark` to clauses. On the other hand, years ago we had the discussion whether a preposition modifying a verb can be `case` or should it be `mark` or even `SCONJ/mark`, but I think this discussion did not result in any strict rule.

But even if clauses require `mark`, you can have an adpositional phrase as a predicate, then the adposition modifies a predicate nominal and not the whole clause, meaning that it can keep `case` as its relation type. Perhaps it is the case here?

Reply:
The word "som" is tricky in the Scandinavian languages. It has a number of distinct uses in relative clauses, comparative constructions, etc. and it is normally regarded as a conjunction (or in some cases pronoun) rather than an adposition. And it can definitely be combined with adjectival predicates as well as nominal ones, in constructions analogous with "they identified him as guilty" vs. "they identified him as the culprit".

Reply:
OK, so maybe it could be `SCONJ` rather than `ADP` (and BTW the feature `PartType=Inf` seems off to me) but that would require a broader change, as the present version of the treebank [only uses ADP, ADV or PRON for this word](http://hdl.handle.net/11346/PMLTQ-WJIR).

`SCONJ` does not necessarily imply `mark`. In [comparative constructions](https://universaldependencies.org/workgroups/comparatives.html), various languages use a `SCONJ` to mark the standard of comparison, but it is attached as `case` _if the standard of comparison is just a nominal._ So we are back to the question whether _som_ marks a nominal or a clause. I thought I'd look at the other Scandinavian treebanks how they solve similar constructions, but it did not give a clear answer either:

- [Talbanken](http://hdl.handle.net/11346/PMLTQ-FYGD) prefers `SCONJ` + `mark` (with `xcomp` or `advcl` for the noun), the third most frequent situation is `ADP` + `case` + `obl` for the noun. (On the query page, click on "without Filters" to see the actual trees.)
- [Bokmaal](http://hdl.handle.net/11346/PMLTQ-P8QU) seems to be less consistent but the most frequent triple is `ADP` + `case` + `xcomp` for the noun.

Reply:
For now I have changed it to xcomp as is it both the minimal change while being consistent with talbanken and bokmaal. Haven't changed anything related to 'som' (yet)